tests/foreman/rhai/test_rhai.py

```diff
@@ -80,7 +79,7 @@ def test_client_registration_to_rhai(self):
             vm.get('/var/log/redhat-access-insights/redhat-access'
                    '-insights.log', './insights_client_registration.log')
 
-    def test_org_selection_for_rhai(self):
+    def test_negative_org_not_selected(self):
```

Comment:
s/test_negative_org_not_selected/test_negative_org_selection ??
negative+not = postive :)